### PR TITLE
Add Tests for report_generator.get image and update project version

### LINKED ISSUE #24: Create tests/test_report_generator.py test_get_image()

### LINKED ISSUE #63: Update Version and Release (1.0.0)
We need to update the project version from the initial skeleton version "0.0.1" to a major release version of "1.0.0". Once this issue is completed, any new development should follow the pre/post merge versioning standards process. See https://github.com/cisagov/tpt-reports/pull/51#issuecomment-1737566932 for recommendations and details around versioning.

**Since there is currently no baseline version and it is not recommended to submit a PR exclusively for version changes, we will be working this into the next PR for improvements.

### PR DESCRIPTION
**Pull request by nickviola**
#  #

## 🗣 Description ##
Adding unit tests for report_generator.get_image (Closes #24). This PR also includes work to update the project version (Closes #63). Created an initial "1.0.0" major release as a baseline and then stacked a prerelease to capture the changes in this PR.

## 💭 Motivation and context ##
Adding test coverage and updating version number of project. Establishes a versioning baseline for us to modify in future PRs.

## 🧪 Testing ##
Run unit tests in the project and confirm version is updated to 1.0.0 with additional prerelease for code changes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: tests/test_report_generator.py [hunk 1]
```diff
@@ -86,3 +87,41 @@ def test_report_gen(test_dictionary):
         assert result.filename == output_file_path
         # Validate the page size of result
         assert result.pagesize == (612.0, 792.0)
+
+
+@patch("tpt_reports.report_generator.utils.ImageReader")
+@patch("tpt_reports.report_generator.Image")
+class TestGetImage(TestCase):
+    """Test get_image clase."""
```
Comment (dav3r) [inline tests/test_report_generator.py]: Fix typo and improve wording:
```suggestion
    """Tests for the get_image function."""
```

File: tests/test_report_generator.py [hunk 2]
```diff
@@ -86,3 +87,41 @@ def test_report_gen(test_dictionary):
...
+
+    def test_get_image_invalid_params(self, mock_image, mock_image_reader):
+        """Test that the get_image function raises an exception invalid args."""
```
Comment (dav3r) [inline tests/test_report_generator.py]: Improve verbiage:
```suggestion
        """Test that get_image raises an exception when given invalid arguments."""
```

**Review by dav3r — APPROVED** [review]
LGTM, aside from a couple of wording quibbles. 👍

**Review by schmelz21 — APPROVED** [review]
approved

**Pull request merged**